Sources/PointFree/Account/Account.swift

```diff
@@ -10,7 +10,7 @@ import Styleguide
 import Tuple
 
 let accountResponse =
-  requireUser
+  _requireUser
```

Comment:
Renamed the "bad" `requireUser` to `_requireUser` and made `requireUser` handle `T{N}` instead of `Tuple{N}`. I think we'll wanna change to use the `T{N}` system everywhere eventually.